common/isolationgroup/interface.go

```diff
@@ -50,16 +47,8 @@ type State interface {
 
 	Subscribe(domainID, key string, notifyChannel chan<- ChangeEvent) error
 	Unsubscribe(domainID, key string) error
-}
 
-// Config values for the partitioning library for segmenting portions of workflows into isolation-groups - a resiliency
-// concept meant to help move workflows around and away from failure zones.
-type Config struct {
```

Comment:
unexposing this and moving it to the default implmentation since there's no need for it to be exposed